From 899d1f785319971a522a7d9c111b9bbcb75445fc Mon Sep 17 00:00:00 2001
From: Sergio Costas <raster@rastersoft.com>
Date: Sat, 15 Dec 2018 23:29:22 +0100
Subject: [PATCH 53/55] desktopManager.js: fix typo in destroy deleteChildrenId

Due to a typo, the destroy() method checked _destroyChildrenId instead
of _deleteChildrenId. This patch fixes it.
---
 desktopManager.js | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/desktopManager.js b/desktopManager.js
index 9175672..a8802b6 100644
--- a/desktopManager.js
+++ b/desktopManager.js
@@ -693,9 +693,9 @@ var DesktopManager = GObject.registerClass({
             GLib.source_remove(this._layoutChildrenId);
         this._layoutChildrenId = 0;
 
-        if (this._destroyChildrenId)
-            GLib.source_remove(this._destroyChildrenId);
-        this._destroyChildrenId = 0;
+        if (this._deleteChildrenId)
+            GLib.source_remove(this._deleteChildrenId);
+        this._deleteChildrenId = 0;
 
         if (this._monitorsChangedId)
             Main.layoutManager.disconnect(this._monitorsChangedId);
-- 
2.20.0

